cypress/fixtures/stubbing/ScenarioSharing/workspaces.js

```diff
@@ -18,9 +19,25 @@ export const WORKSPACE_NO_USERS = {
   id: 'W-stbbdbwry0',
   key: 'DemoBreweryNoUsers',
   security: {
-    default: '',
+    default: ROLES.SCENARIO.VIEWER,
     accessControlList: [],
   },
 };
 
-export const WORKSPACES_LIST = [WORKSPACE_NO_USERS];
+const usersAccess = [
+  { id: USERS_LIST[0].email, role: ROLES.SCENARIO.ADMIN },
+  { id: USERS_LIST[1].email, role: ROLES.SCENARIO.VIEWER },
+  { id: USERS_LIST[2].email, role: ROLES.SCENARIO.VIEWER },
+  { id: USERS_LIST[3].email, role: ROLES.SCENARIO.VIEWER },
+];
+export const WORKSPACE_WITH_USERS_LIST = {
+  ...WORKSPACE_EXAMPLE,
+  id: 'W-stbbdbwsx1',
+  key: 'DemoBreweryNoUsers',
+  security: {
+    default: ROLES.SCENARIO.VIEWER,
```

Comment:
If we want to check the accessControlList, we should probably set 'none' as the default security